{
    "affected": [
        {
            "package": {
                "ecosystem": "photon:2.0",
                "name": "PyYAML",
                "purl": "pkg:rpm/vmware/PyYAML?distro=photon-2"
            }
        },
        {
            "package": {
                "ecosystem": "photon:2.0",
                "name": "envoy",
                "purl": "pkg:rpm/vmware/envoy?distro=photon-2"
            }
        },
        {
            "package": {
                "ecosystem": "photon:2.0",
                "name": "kubernetes",
                "purl": "pkg:rpm/vmware/kubernetes?distro=photon-2"
            }
        },
        {
            "package": {
                "ecosystem": "photon:2.0",
                "name": "zsh",
                "purl": "pkg:rpm/vmware/zsh?distro=photon-2"
            }
        },
        {
            "package": {
                "ecosystem": "photon:2.0",
                "name": "gnupg",
                "purl": "pkg:rpm/vmware/gnupg?distro=photon-2"
            }
        }
    ],
    "id": "PHSA-2020-0229",
    "modified": "2023-03-16T05:25:30Z",
    "published": "2020-04-14T00:00:00Z",
    "references": [
        {
            "type": "ADVISORY",
            "url": "https://github.com/vmware/photon/wiki/Security-Update-2.0-229"
        }
    ],
    "related": [
        "CVE-2020-8659",
        "CVE-2020-8664",
        "CVE-2019-14855",
        "CVE-2019-18838",
        "CVE-2020-8552",
        "CVE-2019-18836",
        "CVE-2019-11250",
        "CVE-2019-20044",
        "CVE-2020-1747",
        "CVE-2019-15225",
        "CVE-2020-8661",
        "CVE-2019-11251",
        "CVE-2019-15226"
    ]
}